quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/ConfigCategory.java

```diff
@@ -2,15 +2,15 @@
 
 public enum ConfigCategory {
     // ordered by name asc
-    CLUSTERING("Cluster:", 10),
-    DATABASE("Database:", 20),
-    FEATURE("Feature:", 30),
-    HOSTNAME("Hostname:", 40),
-    HTTP("HTTP/TLS:", 50),
-    METRICS("Metrics:", 60),
-    PROXY("Proxy:", 70),
-    VAULT("Vault:", 80),
-    GENERAL("General:", 999);
+    CLUSTERING("Cluster", 10),
```

Comment:
I think this change will require updating some text files used by tests to assert the help message,